Formula/g/gnupg.rb

```diff
@@ -39,21 +51,28 @@ class Gnupg < Formula
     depends_on "gettext"
   end
 
-  # Backport fix for missing unistd.h
-  patch do
-    url "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=patch;h=1d5cfa9b7fd22e1c46eeed5fa9fed2af6f81d34f"
-    sha256 "610d0c50004e900f1310f58255fbf559db641edf22abb86a6f0eb6c270959a5d"
-  end
-
   def install
     libusb = Formula["libusb"]
     ENV.append "CPPFLAGS", "-I#{libusb.opt_include}/libusb-#{libusb.version.major_minor}"
+    ENV.append "LDFLAGS", "-L#{libusb.opt_lib}"
+
+    readline = Formula["readline"]
+    ENV.append "CPPFLAGS", "-I#{readline.opt_include}"
+    ENV.append "LDFLAGS", "-L#{readline.opt_lib}"
```

Comment:
Are these things reported upstream, or a patch submitted or anything?

Reply:
These are not to be reported upstream.
`readline` is keg-only, so such a handling is required.
`libusb` installs headers in an unusual place, libs are also added just for the sake of completeness and symmetry

Reply:
Dear @SMillerDev 
Please, point to the policy that requires me to DISABLE something at the date of EOL.
Given you know the policy as a maintainer, you could point me toward the correct solution.
Instead, you're harassing an external contributor.
Cc: @p-linnane @MikeMcQuaid 

The current precedent is set by @carlocab in `gnupg@2.2` by giving 6-month grace period before disabling the formula.
Disabling is a drastic action, but giving 6 months grace is kind of okay.
See: https://github.com/Homebrew/homebrew-core/commit/fc02e617a138675f6b133f5a4ce97eb2e0fd9c09

I've updated the PR by both deprecating at the date of EOL and disabling 6 months later.

This is as much as I can agree. No further conscious action is possible by my side.
If you, @SMillerDev, don't comply, I just go into private tap.